src/js/editor.js

```diff
@@ -203,6 +205,8 @@ class ToastUIEditor {
 
     this.setValue(this.options.initialValue, false);
 
+    this.setInitialHtml(this.initialHtml);
```

Comment:
I think `options.initialValue` is the one that user explicitly set over initial HTML value from DOM which is implicit.

Intial HTML value from DOM should not overrides `options.initialValue`.

Reply:
Oh, I couldn't consider about that. Thank you!